chart/k8gb/templates/operator.yaml

```diff
@@ -121,3 +121,5 @@ spec:
               value: {{ quote .Values.k8gb.log.level }}
             - name: NO_COLOR
               value: "true"
+            - name: SPLIT_BRAIN_PREVENTION
+              value: {{ quote .Values.k8gb.noSplitBrain }}
```

Comment:
Chart value name is contraversial with env.

I suggest: `noSplitBrainDetection`

Reply:
or diableSplitBrainDetection

Reply:
@k0da +1, no sure though why we need to use `no-` or `disable-` prefix for variables, we can simply set the default value to `false`, like already done here: https://github.com/AbsaOSS/k8gb/pull/465/files#diff-b94e4b6c67fca17dce52708f765679ec46a8553531b432d30b8da1b11b1bc89cR84.
I'm also fine with either `splitBrainDetection` or `splitBrainPrevention`, it's just should be named consistently over the codebase.

Reply:
Can we have just `splitBrain` true/false?

Reply:
@ytsarev, as I mentioned earlier, split-brain is the name of the issue happening in the distributed system. So if we tell that we want to enable or disable the split-brain - this looks exactly confusing from user point of view

Reply:
It would be more correct to tell that we want to enable or disable the split-brain detection or prevention feature. We don't want to enable the problem, we want to enable the solution.

Reply:
splitBrainCheck true/false

Reply:
@somaritane, @k0da  
`SPLIT_BRAIN_CHECK` 
  - default value (empty or not set) false
  - invalid value - false
  - true, t, 1,T ,True, ...  - enables split-brain otherwise it's disabled 

If we do it this way, then enabling split-brain will require to set `SPLIT_BRAIN_CHECK` to true on each k8gb you want to have split-brain, otherwise it will be disabled.

Reply:
👍

Reply:
@kuritka looks good 👍  , let's also update the PR title and body for consistency with amendments.